src/tokens.ts

```diff
@@ -0,0 +1,40 @@
+/*
+  Copyright (c) 2022 MarketSquare
+  Distributed under the terms of the BSD-3-Clause License
+*/
+
+/** The npm namespace */
+export const NS = '@marketsquare/jupyterlab_robotmode';
+
+/** The source of truth for the mode plugin id */
+export const MODE_PLUGIN_ID = `${NS}:plugin`;
+
+/** The source of truth for the robot file plugin id */
+export const FILE_PLUGIN_ID = `${NS}:robot-file-plugin`;
+
+/** The source of truth for the resource file plugin id */
+export const RESOURCE_PLUGIN_ID = `${NS}:robot-resource-plugin`;
+
+/** well-known mime type for robot framework (pygments, etc.) */
+export const MIME_TYPE = 'text/x-robotframework';
+
+/** the canonical CodeMirror mode name */
+export const MODE_NAME = 'robotframework';
+
+/** the human-readable name of the CodeMirror mode */
+export const MODE_LABEL = 'Robot Framework';
```

Comment:
@bollwyvl I have no idea why, but this works for Robot Kernel notebooks only when `MODE_LABEL = 'robotframework';`

Reply:
From RobotKernel:

```
    language = "robotframework"
    language_info = {
        "mimetype": "text/x-robotframework",
        "name": "Robot Framework",
        "file_extension": ".robot",
        "codemirror_mode": "robotframework",
        "pygments_lexer": "robotframework",
    }
 ```

Reply:
So, this is probably just me not knowing what I was doing years ago. Maybe I could change this in robotkernel 1.6.0 and release it just after robotmode 0.4.0. Technically breaking changes are allowed in 0.x -releases :see_no_evil:

Reply:
Oh, reasoning for that goes back to Classic Notebook. What ever kernel announces as its `codemirror_mode`, Classic Notebook tries to load from Code Mirror.